google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/CollectionReference.java

```diff
@@ -167,6 +167,22 @@ public void remove() {
     };
   }
 
+  @Nonnull
+  public ApiFuture<?> futureListDocuments() {
+    ListDocumentsRequest.Builder request = ListDocumentsRequest.newBuilder();
+    request.setParent(options.getParentPath().toString());
+    request.setCollectionId(options.getCollectionId());
+    request.setMask(DocumentMask.getDefaultInstance());
+    request.setShowMissing(true);
+
+    try {
+      return firestore.sendRequest(
+          request.build(), firestore.getClient().listDocumentsPagedCallable());
```

Comment:
Is there any other callable present( imean other than `firestore.getClient().listDocumentsPagedCallable()` ?

I am just curious as this is `listDocumentsPagedCallable()`.. there could be just `listDocuemntsCallable()`

Reply:
Make sure if callable() vs pagedCallable() response delay.. if pagedCallable is delaying in between then its better to use `listDocuemntsCallable()`.. (ofcourse if its available).

Reply:
> Is there any other callable present( imean other than `firestore.getClient().listDocumentsPagedCallable()` ?
> 
> I am just curious as this is `listDocumentsPagedCallable()`.. there could be just `listDocuemntsCallable()`

There is no such type of callable(listDocuemntsCallable()) present.